__tests__/sanity.test.js

```diff
@@ -17,3 +17,11 @@ it('generates the right CSS', () => {
       expect(result.css).toBe(expected)
     })
 })
+
+it('Do not change css if tailwind is not used at all', () => {
```

Comment:
One more comment actually! Try to name the tests so that they read properly starting with the word "it". In this case maybe this would be a good name:

"does not add any CSS if no Tailwind features are used"

Reply:
done and thanks for the input :-)